content/metadata.yaml

```diff
@@ -567,6 +567,26 @@ authors:
         contributions:
           - Writing - Review & Editing
           - Writing - Original Draft
+  -
+    github: vincerubinetti
+    name: Vincent Rubinetti
+    initials: VR
+    orcid: 0000-0002-4655-3773
+    email: vince.rubinetti@gmail.com
+    contributions:
+      - Writing - Original Draft
+    code of conduct:
+      confirmed: No
```

Comment:
If you want to approve, here's the link: https://github.com/greenelab/covid19-review/blob/master/CODE_OF_CONDUCT.md